engine/runner/src/main/scala/org/enso/runner/LanguageServerApp.scala

```diff
@@ -20,11 +19,11 @@ object LanguageServerApp {
     * @param logLevel log level
     */
   def run(config: LanguageServerConfig, logLevel: LogLevel): Unit = {
-    println("Starting Language Server...")
     val server = new LanguageServerComponent(config, logLevel)
-    Await.result(server.start(), 10.seconds)
-    StdIn.readLine()
```

Comment:
This will break the Project Manager as it relies on using newline to shutdown the language server.

We discussed it with @4e6, as he initially suggested using a message instead, but the newline is more robust in the sense that it also works if the websocket to send the message to fails to start.

Maybe you can handle this shutdown logic differently depending on some `--daemon` flag?

Alternatively if you register a shutdown hook, maybe the PM can just send a SIGTERM signal to the LS which should trigger the hook if I understand correctly?

Anyway, the Project Manager should be able to shutdown the Language Server, currently the tests seem to indicate that this doesn't work:
```
[2020-12-15T15:46:00.087Z] [org.enso.projectmanager.infrastructure.languageserver.LanguageServerController] Language server shutdown timed out
```

Reply:
I decided to add daemon flag.